Nucleus/Nucleus.Rhino/RhinoOutput.cs

```diff
@@ -20,10 +20,25 @@ namespace Nucleus.Rhino
     public static class RhinoOutput
     {
         /// <summary>
-        /// Is Salamander currently writing to the Rhino document?
+        /// Is tool currently writing to the Rhino document?
         /// </summary>
         public static bool Writing { get; set; } = false;
 
+        /// <summary>
+        /// Get current active document as default
+        /// </summary>
+        private static RhinoDoc ActiveRhinoDoc { get; set; } = RhinoDoc.ActiveDoc;
```

Comment:
The ActiveDoc in Rhino will change.  So, this should be modified to have a private backing field which, if null, will return RhinoDoc.ActiveDoc in the getter instead.  Otherwise this would probably break everything if the user opened a new document in Rhino.